prow/external-plugins/cherrypicker/server.go

```diff
@@ -361,7 +361,9 @@ var cherryPickBranchFmt = "cherry-pick-%d-to-%s"
 
 func (s *Server) handle(l *logrus.Entry, requestor string, comment *github.IssueComment, org, repo, targetBranch, title, body string, num int) error {
 	if err := s.ensureForkExists(org, repo); err != nil {
-		return err
+		resp := fmt.Sprintf("cannot fork %s/%s: %v", org, repo, err)
+		s.log.WithFields(l.Data).Info(resp)
```

Comment:
Why not just use `l` as the logger and instead injects its fields into the main logger?

Reply:
Copy-pasta from another log line /shrug